tests/compatibility-tests/pom.xml

```diff
@@ -265,6 +271,39 @@
       </plugins>
    </build>
    <profiles>
+      <profile>
+         <id>compatibility-real-servers</id>
+         <properties>
+            <skipCompatibilityTests>false</skipCompatibilityTests>
```

Comment:
This will also enable all the other tests to be picked up by surefire even though their deps may not have been prepared if the other profile isnt run...do they all handle that (I see the new test handles the reverse case with an assumptions check) or is it just the case that you _must_ run both profiles if you run the new one?

EDIT: the answer is that the existing tests do not handle it, you need both. Should we make it clearer (e.g a comment here) that they both need to be enabled? E.g instead of setting this true we could have the new profile enforce the skipCompatibilityTests prop is true, and indicate the other profile needs to be used if it isnt.

Reply:
Yeah.. I'm removing that.. it was a mistake